### PR TITLE
Cassandra fix flaky test

### LINKED ISSUE #8788: Flaky test TestCassandraTokenSplitManager
```
    Tests run: 749, Failures: 1, Errors: 0, Skipped: 7, Time elapsed: 465.517 sec <<< FAILURE! - in TestSuite
    testNonEmptyTable(com.facebook.presto.cassandra.TestCassandraTokenSplitManager)  Time elapsed: 0.309 sec  <<< FAILURE!
    java.lang.AssertionError: expected [10] but found [1]
        at com.facebook.presto.cassandra.TestCassandraTokenSplitManager.testNonEmptyTable(TestCassandraTokenSplitManager.java:69)
```

### PR DESCRIPTION
**Pull request by arhimondr**
Fixes #8788

### CODE REVIEW

File: presto-cassandra/src/test/java/com/facebook/presto/cassandra/EmbeddedCassandra.java [hunk 1]
```diff
@@ -167,6 +168,7 @@ public static void refreshSizeEstimates(String keyspace, String table)
                 return;
             }
             log.info("Size estimates haven't been refreshed as expected. Retrying ...");
+            SECONDS.sleep(1);
```
Comment (cawallin) [inline presto-cassandra/src/test/java/com/facebook/presto/cassandra/EmbeddedCassandra.java]: Do you see this making the tests significantly longer? 500ms might be better.

File: presto-cassandra/src/test/java/com/facebook/presto/cassandra/TestCassandraTokenSplitManager.java [hunk 2]
```diff
@@ -25,9 +25,9 @@
 
 public class TestCassandraTokenSplitManager
 {
-    private static final int SPLIT_SIZE = 100;
+    private static final int SPLIT_SIZE = 256;
     private static final String KEYSPACE = "test_cassandra_token_split_manager_keyspace";
-    private static final int PARTITION_COUNT = 1000;
+    private static final int PARTITION_COUNT = 2048;
```
Comment (cawallin) [inline presto-cassandra/src/test/java/com/facebook/presto/cassandra/TestCassandraTokenSplitManager.java]: Is this change necessary? Does this mean the `CassandraTokenSplitManager`has issues with non-power-of-two partition counts?

Comment (arhimondr) [inline presto-cassandra/src/test/java/com/facebook/presto/cassandra/TestCassandraTokenSplitManager.java]: It always rounds to `% 1024`. For 1000 partitions it shows that there are 1024 partitions. I can set these constants to `1024` and `128` though.

**Comment by cawallin** [comment]
Merged, thanks!

**Pull request merged**